styles.css

```diff
@@ -1,582 +1,600 @@
-  list-style: none;
-  align-self: center;
-  padding-left: 0;
-}
-
-.diamond {
-  margin-left: 80px;
-  width: 18px;
-  height: 18px;
-  background-color: #ff6b00;
-  transform: rotate(45deg);
-  display: inline-block;
-}
-
-.square {
-  margin-left: 60px;
-  width: 18px;
-  height: 18px;
-  background-color: #ff6b00;
-  display: inline-block;
-}
-
-.circle {
-  margin-left: 120px;
-  width: 18px;
-  height: 18px;
-  border-radius: 9px;
-  background-color: #ff6b00;
-  display: inline-block;
-}
-
-.forms {
-  position: absolute;
-  margin-top: 6300px;
-  width: 327px;
-  left: 0;
-  padding: 120px 24px;
-  display: flex;
-  flex-direction: column;
-}
-
-.forms h2 {
-  order: 0;
-  align-self: center;
-  margin-bottom: 58px;
-  text-align: center;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-  color: #172b4d;
-}
-
-.forms form {
-  order: 1;
-  align-self: left;
-  display: flex;
-  height: 330px;
-  flex-direction: column;
-  gap: 24px;
-}
-
-.forms form input {
-  justify-self: stretch;
-  padding: 15px 16px;
-  height: 48px;
-  border-style: none;
-  border-bottom-style: solid;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: rgb(202, 198, 198);
-}
-
-.forms form #msg {
-  min-height: 114px;
-  max-height: 114px;
-  background-color: #fbf8f7;
-  border-style: none;
-  padding: 12px;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: rgb(202, 198, 198);
-}
-
-.forms button {
-  width: 131px;
-  height: 48px;
-  background-color: #ff6b00;
-  padding: 12px;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 17px;
-  line-height: 24px;
-  color: #fff;
-  border: none;
-  align-self: center;
-}
-
-.forms button:hover {
-  transition: 0.25s;
-  background-color: #fcc49b;
-  color: #ff6b00;
-  text-shadow: none;
-  border: 1px solid #ff6b00;
-}
-
-.forms button:disabled {
-  transition: 0.25s;
-  background-color: rgb(192, 192, 192);
-  color: rgb(92, 92, 92);
-  text-shadow: none;
-  border: 1px solid grey;
-}
-
-.forms button:active {
-  background-color: #ff6b00;
-  color: #fff;
-  text-shadow: none;
-  border: 1px solid #ff6b00;
-}
-
-footer {
-  position: absolute;
-  margin-top: 7180px;
-  width: 375px;
-  left: 0;
-  padding: 0;
-  border-style: none;
-  border-top-style: solid;
-  border-color: rgb(202, 198, 198);
-  border-width: 2px;
-  display: flex;
-  flex-direction: column;
-}
-
-footer .social-footer {
-  order: 0;
-  align-self: center;
-  display: flex;
-  justify-content: space-between;
-  height: 30px;
-  width: 200px;
-  list-style: none;
-  padding-left: 0;
-}
-
-.rounded-line {
-  width: 134px;
-  height: 5px;
-  background-color: black;
-  border-radius: 5px;
-  align-self: center;
-  margin-bottom: 8px;
+@media (max-width: 680px) {
```

Comment:
- [ ] [OPTIONAL] Please remove these media query code because all the code in this file is going to be applied for a viewport less than `768px`. Or you can change the size to 768px because that is what you used for the desktop version. This created a gap in the viewport size where no CSS is applied. 

![Screen Shot 2022-07-08 at 18 51 25](https://user-images.githubusercontent.com/12524453/178027142-88d050bb-2fcc-44c2-907c-9938a363dbac.png)